### PR TITLE
feat(perf-issues): init consec http detector for dry run

### PR DESCRIPTION
**Pull request by DominikB2014**
Just running a new detector in dry run and not gonna create issues, a lot of detector code is copied from consecutive http.
This detector just looks for at least 5 consecutive http spans, that are at least 1s long.

### CODE REVIEW

File: src/sentry/issues/grouptype.py [hunk 1]
```diff
@@ -115,6 +115,15 @@ class PerformanceFileIOMainThreadGroupType(GroupType):
     category = GroupCategory.PERFORMANCE.value
 
 
+@dataclass(frozen=True)
+class PerformanceConsecutiveHTTPQueriesGroupType(GroupType):
+    type_id = 1009
```
Comment (k-fish) [inline src/sentry/issues/grouptype.py:120]: `1009` wasn't already used? 🤔

**Review by k-fish — APPROVED** [review]
You can get this to pass by adding your new type to the 'tags' in the metrics reporting test, or better yet by removing the enumeration of types as tags and just looking at the unc. asset tag.

File: tests/sentry/utils/performance_issues/test_consecutive_http_detector.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+)
+from sentry.testutils.silo import region_silo_test
+from sentry.utils.performance_issues.detectors import ConsecutiveHTTPSpanDetector
+from sentry.utils.performance_issues.performance_detection import (
+    PerformanceProblem,
+    get_detection_settings,
+    run_detector_on_data,
+)
+
+SECOND = 1000
```
[the remark this answers is not in the document]
Comment (DominikB2014) [reply tests/sentry/utils/performance_issues/test_consecutive_http_detector.py:21]: I guess my IDE didn't catch the unused variable haha, its not used.
Thanks, good catch!

**Comment by DominikB2014** [comment]
> You can get this to pass by adding your new type to the 'tags' in the metrics reporting test, or better yet by removing the enumeration of types as tags and just looking at the unc. asset tag.

Thanks! Yeah I added it and confirmed it fixed it.
I do like the idea of just looking at. unc asset tag and i'll look into that another PR, i'm not sure if there's a way to do something like `expect.objectContaining` but in pytest.

**Pull request merged**